src/xcompact3d.f90

```diff
@@ -273,11 +272,29 @@ subroutine init_xcompact3d()
   if (itype==2) then
      if(nrank.eq.0)then
         open(42,file='time_evol.dat',form='formatted')
+        if(mhd_active) then
+           open(43,file='mhd_time_evol.dat',form='formatted')
+        endif
+     endif
+  endif
+
+  if (itype==3) then
+     if (nrank.eq.0)then
+        open(52,file='chan_time_evol.dat',form='formatted')
+     endif
+  endif
+  
+  if (itype==5) then
```

Comment:
This is not equivalent to the current code.

Reply:
I have removed the `itype==5` if statement